app/views/practical_supports/_edit.html.erb

```diff
@@ -1,65 +1,47 @@
-<div class="row info-form practical-support-item" id="practical-support-item-<%= practical_support.id %>">
-  <%= bootstrap_form_with model: practical_support,
-                         url: patient_practical_support_path(patient, practical_support),
-                         html: { class: 'edit_practical_support col-12' },
-                         local: false do |f| %>
-    <div class="row">
-      <div class="col-3">
-        <strong><%= f.static_control :support_type, hide_label: true %></strong>
-      </div>
-
-      <div class="col-5">
-        <%= f.select :source,
-                     options_for_select(practical_support_source_options(practical_support.source),
-                                        practical_support.source),
-                     hide_label: true %>
-      </div>
-
-      <div class="col-4">
-        <%= f.number_field :amount,
-                           value: number_to_currency(f.object.amount.to_f, unit: '', delimiter: ''),
-                           hide_label: true,
-                           autocomplete: 'off',
-                           prepend: '$',
-                           step: '0.01' %>
-      </div>
-
-      <div class="col-3">
-        <%= f.date_field :support_date, autocomplete: 'off', hide_label: true %>
-      </div>
-
+<div class="row">
+  <div class="col">
+    <%= bootstrap_form_with model: support,
```

Comment:
any particular reason we renamed the model here? Just shorter and easier?

Reply:
pretty much yeah. Less to type!